manifests/moco-nodes.pp

```diff
@@ -383,6 +383,8 @@
     $beetmoverworker_env = 'mobile-staging'
     $timezone                 = 'UTC'
     $only_user_ssh            = true
+    $pin_puppet_server = 'releng-puppet2.srv.releng.mdc1.mozilla.com'
```

Comment:
For the record, this is temporary and will go away before merging. I just use this as it's easier to edit on my local machine and pull in my puppet environment.